go-controller/pkg/ovn/egressip_test.go

```diff
@@ -293,6 +293,21 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 					gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 					expectedNatLogicalPort := "k8s-node2"
+					egressIPServedAS, _ := addressset.GetDbObjsForAS(
```

Comment:
maybe we can make some functions to improve readability? like
```
getEIPAddrSet(name egressIpAddrSetName, ip string) (*nbdb.AddressSet, *nbdb.AddressSet) {
   v4, v6 := addressset.GetDbObjsForAS(getEgressIPAddrSetDbIDs(name, DefaultNetworkControllerName),
		[]net.IP{net.ParseIP(ip)})
  v4.UUID = v4.Name + "UUID"
  v6.UUID = v6.Name + "UUID"
 return v4, v6
}
```
and a couple more for different types of address sets?

Reply:
done